Firestore/core/src/firebase/firestore/remote/datastore.mm

```diff
@@ -16,14 +16,18 @@
 
 #include "Firestore/core/src/firebase/firestore/remote/datastore.h"
 
+#include "Firestore/core/src/firebase/firestore/util/hard_assert.h"
+
 namespace firebase {
 namespace firestore {
 namespace remote {
 
-Datastore::Datastore() {
-}
-
-Datastore::~Datastore() {
+FirestoreErrorCode Datastore::ToFirestoreErrorCode(
+    grpc::StatusCode grpc_error) {
+  HARD_ASSERT(
+      grpc_error >= grpc::CANCELLED && grpc_error <= grpc::UNAUTHENTICATED,
```

Comment:
grpc::OK (=0) might also be a valid code, I guess. Why start from grpc::CANCELLED (=1) ?

Reply:
Per Rich's offline comment, I changed this method to instead convert between `grpc::Status` and `util::Status`, so now this should be less counter-intuitive.